docs/environment.md

```diff
@@ -12,28 +12,85 @@ You have two options for working with our codebase:
 submodule. We only recommended that if you are very handy with Python. More details about this option
 [are here](innereye_as_submodule.md).
 
-## Windows Subsystem for Linux Setup
+### Set Up Windows Subsystem for Linux
 When developing on a Windows machine, we recommend using [the Windows Subsystem for Linux, WSL2](https://docs.microsoft.com/en-us/windows/wsl/about).
 That's because PyTorch has better support for Linux.  If you want to use WSL2, please follow
 [these instructions](/docs/WSL.md) , that correspond to the manual installation in the official docs.
 
-## Installing Conda or Miniconda
+### Install Conda or Miniconda
 You can skip this step if you have installed WSL as per the previous item.
 
 Download a Conda or Miniconda [installer for your platform](https://docs.conda.io/en/latest/miniconda.html)
 and run it.
 
-## Creating a Conda environment
-Note that in order to create the Conda environment you will need to have build tools installed on your machine. If you are running Windows, they should be already installed with Conda distribution.
+## Create a Conda Environment
+
+There are three important configuration files in this repo for creating conda environments:
+- **`primary_deps.yml`** - This file contains the list of primary package dependencies, and can be used to create an environment on any OS.
+- **`environment.yml`** - **DO NOT EDIT THIS FILE MANUALLY**. This file contains a locked list of primary and secondary dependencies that is used to create the environments for your AzureML jobs and local Linux deployments. As such it contains linux-specific platform dependencies (see [Set Up WSL](#set-up-windows-subsystem-for-linux)) and cannot be used to create a windows environment.
+- **`environment_win.yml`** - This file contains a windows-dependent list of primary and secondary dependencies. This is the file used to create a locked python environment on windows machines.
+
+### Create environment from lockfile
+
+To create an environment from one of the lockfiles, run the following command:
+
+```conda env create --file <environment.yml/environment_win.yml>```
+
+Selecting the relevant file depending on your OS platform (`environment.yml` for Linux, `environment_win.yml` for windows). This will create a new conda environment which can be activated by running:
```

Comment:
```suggestion
Select the relevant file depending on your OS platform (`environment.yml` for Linux, `environment_win.yml` for Windows). This will create a new Conda environment which can be activated by running:
```